samples/SocialSample/Startup.cs

```diff
@@ -135,16 +135,16 @@ 127.0.0.1 MsSecSample.localhost.this
                 options.ClaimsIssuer = "OAuth2-Github";
                 options.SaveTokensAsClaims = false;
                 // Retrieving user information is unique to each provider.
-                options.Notifications = new OAuthAuthenticationNotifications
+                options.Events = new OAuthAuthenticationEvents
```

Comment:
Well in that one we were changing the UseOAuthAuthentication method (and leaving the classes alone).  I'm actually suggesting the opposite now, where we leave the Use/Adds alone, and clean up all of the types instead.

Cookie/Facebook/Google/MicrosoftAccount/OIDC/JwtBearer [Authentication] is already in the namespace that they included to pickup the class no?  For example Mvc doesn't prefix all of their classes with Mvc...